coming-soon.html

```diff
@@ -0,0 +1,121 @@
+        </div>
+        <div class="header-filter">
+            <div class="header-filter-options">
+                <button type="button" class="drop-button">Select filters</button>
+                <div class="dropdown-select">
+                    <ul>
+                        <li>
+                            <input type="checkbox" value="Action" />
+                            <span>Action</span></li>
+                        <li>
+                            <input type="checkbox" value="Comedy" />
+                            <span>Comedy</span></li>
+                        <li>
+                            <input type="checkbox" value="Horror" />
+                            <span>Horror</span></li>
+                        <li>
+                            <input type="checkbox" value="Crime" />
+                            <span>Crime</span></li>
+                        <li>
+                            <input type="checkbox" value="Thriller" />
+                            <span>Thriller</span></li>
+                        <li>
+                            <input type="checkbox" value="Sci-fi" />
+                            <span>Sci-fi</span></li>
+                        <li>
+                            <input type="checkbox" value="Animated" />
+                            <span>Animated</span></li>
+                        <li>
+                            <input type="checkbox" value="Drama" />
+                            <span>Drama</span></li>
+                        <li>
+                            <input type="checkbox" value="Romance" />
+                            <span>Romance</span></li>
+                        <li>
+                            <input type="checkbox" value="Adventure" />
+                            <span>Adventure</span></li>
+                        <li>
+                            <input type="checkbox" value="Western" />
+                            <span>Western</span></li>
+                    </ul>
+                </div>
+            </div>
+            <button type="button" class="header-filter-button">
+                Apply filter
+            </button>
+        </div>
+        <!-- This is the header section -->
+    </section>
+    <section class="main-body">
+        <div class="nav">
+            <ul>
+                <li>
+                    <a href="index.html">This week</a></li>
+                <li>
+                    <a href="coming-soon.html">Coming Soon</a></li>
+                <li>
+                    <a href="book-ticket.html">Book a ticket</a></li>
+                <li>
+                    <a href="about.html">About the cinema</a></li>
+            </ul>
+        </div>
+        <div class="main-panel">
+            <h2>01 October - 07 October</h2>
+            <div class="movie-row first">
+                <div class="poster-container">
+                    <a class="poster-link" href="more-html/96.html" title="Кликнете тук за страницата на филма 96">
```

Comment:
what about removing  <div class="poster-container">
and having only an link tag:         <a class="poster"

Reply:
Just to change the class name to <a class="poster" ?

Reply:
sorry and remove                 <div class="poster-container">